### PR TITLE
Formula timer 2

### PR DESCRIPTION
**Pull request by nataliemisasi**

### CODE REVIEW

File: help/en/docs/formula-timer.md [hunk 1]
```diff
@@ -3,7 +3,9 @@ Formula Timer
 
 Grist has a built-in formula timer that will measure the time it takes to evaluate each formula in a document. This helps diagnose which formulas are responsible for slow performance when a document is first opened, or when a document responds to changes.
 
-Grist's formula timer can be found on the 'Document Settings' page, under 'Data Engine'. Select 'Start timing' to begin.
+Grist's formula timer can be found on the 'Document Settings' page, under 'Data Engine'.
```
Comment (nbush) [inline help/en/docs/formula-timer.md:6]: I believe the single quotes in this article should be double quotes (based on what I remember and a brief survey of other docs).

Comment (nataliemisasi) [reply help/en/docs/formula-timer.md:6]: UI Text uses single quotes (https://grist.quip.com/ei5kAlQZxj7M/Grist-Content-Style-Guide)

**Review by nbush — APPROVED** [review]
👍

File: help/en/docs/formula-timer.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (nataliemisasi) [reply help/en/docs/formula-timer.md:6]: 'Settings' is the language used in left-hand nav panel but 'Document Settings' is the page name

**Pull request merged**